src/panels/config/repairs/dialog-repairs-issue-subtitle.ts

```diff
@@ -0,0 +1,53 @@
+import { css, html, LitElement } from "lit";
+import { customElement, property } from "lit/decorators";
+import type { HomeAssistant } from "../../../types";
+import { domainToName } from "../../../data/integration";
+import type { RepairsIssue } from "../../../data/repairs";
+
+@customElement("dialog-repairs-issue-subtitle")
+class DialogRepairsIssueSubtitle extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ type: Object }) public issue!: RepairsIssue;
+
+  protected render() {
+    const domainName = domainToName(this.hass.localize, this.issue.domain);
+    const reportedBy = domainName
+      ? ` ⸱ ${this.hass.localize("ui.panel.config.repairs.reported_by", { integration: domainName })}`
+      : "";
+
+    const severity = this.hass.localize(
+      `ui.panel.config.repairs.${this.issue.severity}`
+    );
+
+    return html`
+      <div class="secondary" .title=${`${severity}${reportedBy}`}>
```

Comment:
No need to wrap this in a div, if you need `display: block` or anything set the styling on `:host`

Reply:
Also no need for a `title` if it has the same content as the content of the div